quasar/assetlist.json

```diff
@@ -18,9 +18,9 @@
       ],
       "type_asset": "sdk.coin",
       "base": "uqsr",
-      "name": "Quasar",
-      "display": "qsr",
-      "symbol": "QSR",
+      "name": "Quasar (legacy)",
+      "display": "qsr.legacy",
```

Comment:
This 'display' unit is not defined anywhere, and therefore the exponent is unknown. I recommend changing this back to what it was.

Reply:
Fixed the "display" unit in PR as per feedback